addons/barfiller/statics.lua

```diff
@@ -0,0 +1,195 @@
+--[[	BSD License Disclaimer
+		Copyright © 2015, Morath86
+		All rights reserved.
+
+		Redistribution and use in source and binary forms, with or without
+		modification, are permitted provided that the following conditions are met:
+
+			* Redistributions of source code must retain the above copyright
+			  notice, this list of conditions and the following disclaimer.
+			* Redistributions in binary form must reproduce the above copyright
+			  notice, this list of conditions and the following disclaimer in the
+			  documentation and/or other materials provided with the distribution.
+			* Neither the name of BarFiller nor the
+			  names of its contributors may be used to endorse or promote products
+			  derived from this software without specific prior written permission.
+
+		THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+		ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+		WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+		DISCLAIMED. IN NO EVENT SHALL Morath86 BE LIABLE FOR ANY
+		DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+		(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+		LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+		ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+		(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+		SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+]]
+
+-- Default settings file
+default_settings = {
+	bar_settings = {
+		background = { -- image: Must be the EXACT path to your Windower\Addon folder
+			image = 'C:\\Program Files (x86)\\Windower4\\addons\\barfiller\\data\\bar_bg.png',
+			pos = {
+				x = 164,
+				y = 6
+			},
+			size = {
+				height = 5,
+				width = 472
+			}
+		},
+		foreground = { -- image: Must be the EXACT path to your Windower\Addon folder
+			image = 'C:\\Program Files (x86)\\Windower4\\addons\\barfiller\\data\\bar_fg.png',
+			pos = {
+				x = 166,
+				y = 6
+			},
+			size = {
+				height = 5,
+				width = 1
+			}
+		}
+	}
+}
+
+-- Current Game Resolution
+windowerResX = windower.get_windower_settings().x_res
```

Comment:
The naming convention in code is to use *snake_case*, i.e. all lower cased letters with underscores separating single words in an identifier.